### PR TITLE
{chem}[gfbf/2023a] xtb-IFF v1.1

### PR DESCRIPTION
**Pull request by WilleBell**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by WilleBell** [comment]
Test report by @WilleBell
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node603.hydra.os - Linux Rocky Linux 8.9, x86_64, AMD EPYC 9384X 32-Core Processor, Python 3.6.8
See https://gist.github.com/WilleBell/4f85391d94a0bb12ccaf3a19f1f0eb6e for a full test report.

File: easybuild/easyconfigs/x/xtb-IFF/xtb-IFF-1.1-gfbf-2023a.eb [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+name = 'xtb-IFF'
+version = '1.1'
+
+homepage = 'https://github.com/grimme-lab/xtbiff'
+description = """
+General Intermolecular Force Field based on Tight-Binding Quantum Chemical Calculations. 
+"""
+
+toolchain = {'name': 'gfbf', 'version': '2023a'}
```
Comment (smoors) [inline easybuild/easyconfigs/x/xtb-IFF/xtb-IFF-1.1-gfbf-2023a.eb:11]: shouldn't this use the `SYSTEM` toolchain?

File: easybuild/easyconfigs/x/xtb-IFF/xtb-IFF-1.1-gfbf-2023a.eb [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+source_urls = ['https://github.com/grimme-lab/xtbiff/releases/download/v1.1']
+sources = ['xtbiff.tar.xz']
+checksums = ['6d5dee10ba39ecb1e26374b7bf82e2364d8ebed5b6f35cdce4ced73245c296f5']
+
+dependencies = [
+    ('CREST', '2.12'),
```
Comment (smoors) [inline easybuild/easyconfigs/x/xtb-IFF/xtb-IFF-1.1-gfbf-2023a.eb:18]: `CREST` is not a dep of `xtb-IFF`.
if there is any, it should be the other way around: `xtb-IFF` as a dep of `CREST`

**Comment by smoors** [comment]
@boegelbot: please test @ generoso

**Comment by smoors** [comment]
@boegelbot please test @ jsc-zen3

**Review by smoors — APPROVED** [review]
lgtm

**Comment by smoors** [comment]
Going in, thanks @WilleBell!

**Pull request merged**